packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -129,6 +129,21 @@ describe('DateTimePicker', () => {
     }
   });
 
+  describe('Time Clarification Dialog Disabled', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-dst-disabled');
+      browser.click('input[name="terra-time-hour-input"]');
+      browser.click('input[name="terra-time-minute-input"]');
+      browser.waitForVisible('[class*="time-clarification"]', 1000);
+      browser.click('[class*="button-daylight"]');
+      browser.click('#date-time-picker-toggler');
+    });
+
+    const ignoredDisabledAlly = Object.assign({ 'color-contrast': { enabled: false } }, ignoredA11y);
+    Terra.should.beAccessible({ rules: ignoredDisabledAlly });
+    Terra.should.matchScreenshot('0');
```

Comment:
Could this screenshot be given a better name?

Reply:
What are you thinking?

Reply:
Disabled would be more descriptive but also kind of repetitive. If we had one overlapping describe block for Time Clarification 'disabled' would be a good name. I think `0` would be confusing, so maybe even leaving it blank and having the screenshot called default would be better.

Reply:
Oh, I see! Since we are just taking a single snapshot, we can remove this and leave it as default! I was thinking you are talking about the top name!